ach/src/test/java/com/adyen/checkout/ach/testrepository/TestAddressRepository.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by onurk on 31/1/2023.
+ */
+
+package com.adyen.checkout.ach.testrepository
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.api.model.AddressItem
+import com.adyen.checkout.components.repository.AddressRepository
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import java.util.Locale
+
+/**
+ * Test implementation of [AddressRepository]. This class should never be used except in test code.
+ */
+@RestrictTo(RestrictTo.Scope.TESTS)
+class TestAddressRepository : AddressRepository {
```

Comment:
This is duplicated right? Let's delete this and move the original class from `card` to `ui-core` in the `test` package?